Formula/i/intercept.rb

```diff
@@ -31,16 +31,15 @@ def install
 
     generate_completions_from_executable(bin/"intercept", "completion")
 
-    pkgshare.install "examples"
+    pkgshare.install "playground"
   end
 
   test do
-    cp_r "#{pkgshare}/examples", testpath
-
-    output = shell_output("#{bin}/intercept config -r")
-    assert_match "Config clear", output
-
-    output = shell_output("#{bin}/intercept config -a examples/policy/minimal.yaml")
-    assert_match "New Config created", output
+    cp_r "#{pkgshare}/playground", testpath
+    cd "playground" do
+      output = shell_output("#{bin}/intercept audit --policy policies/test_scan.yaml " \
+                            "--target targets -vvv audit 2>&1")
+      assert_match "Total Policies: 2", output
```

Comment:
```suggestion
      assert_match(/Total\s+Policies:\s+\d+/i, output)
```

This number will presumably change if upstream adds/removes policies from this file, so we may want to match any number rather than relying on it always being 2.

Reply:
I dont think it would be burden to update the test